.github/workflows/main.yaml

```diff
@@ -43,7 +43,7 @@ jobs:
             cmake \
             libsdl2-dev \
             libphysfs-dev \
-            libboost-dev
+            libboost-all-dev
```

Comment:
May be just add libboost-test-dev ist enough.

Reply:
yes, it appears its enough.